content/blog/2019/10/2019-10-21-thinking-about-jenkins-security.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Thinking About Jenkins Security - DevOps World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- security
+- event
+- community
```

Comment:
```suggestion
```

Reply:
I don't understand that suggestion.  Other blog posts ([here](https://github.com/jenkins-infra/jenkins.io/edit/master/content//blog/2019/05/2019-05-30-becoming-contributor-newbie-tickets.adoc) and [here](https://github.com/jenkins-infra/jenkins.io/edit/master/content//blog/2019/10/2019-10-01-hacktoberfest.adoc) and ...) use the same syntax that I used, a left anchored '-' followed by a single space and the word 'community'.  

Can you help me understand what I'm missing from this suggestion?